docs/extending.md

```diff
@@ -195,6 +195,9 @@ this value will also be used as label.
 
 For the label, it's possible to use a dict with translated values.
 
+Beware, translating the values inside of the `get_data`-method and returning just
```

Comment:
```suggestion
Note: If the labels returned from`get_data()` depend on the current user's language, you need to return a `dict` with the language code as keys instead of translating the value yourself. Returning already 
```

Reply:
Done!